### DISCUSSION TITLE
The problem with sending the project to GITHUB

### DISCUSSION THREAD

**Discussion by zartem2001** [community/community#141771] [Other Feature Feedback, Questions, & Ideas]
I can't send a project from IDEA to GITHUB
![Снимок экрана 2024-10-17 131845](https://github.com/user-attachments/assets/ec135e42-ca21-4a18-8b27-4ecd62057323)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @zartem2001 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including relevant details like which tool(s) you are using. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
